analyzers/tests/SonarAnalyzer.UnitTest/Rules/Hotspots/HardcodedIpAddressTest.cs

```diff
@@ -35,6 +38,15 @@ public class HardcodedIpAddressTest
         public void HardcodedIpAddress_CS() =>
             Verifier.VerifyAnalyzer(@"TestCases\Hotspots\HardcodedIpAddress.cs", new CS.HardcodedIpAddress(AnalyzerConfiguration.AlwaysEnabled));
 
+        [TestMethod]
```

Comment:
Please Try to move this test case to the RuleTests.cs file, and run it against all the Rules we have and not just the `HardcodedIpAddress `rule. Also please rename DictionaryStressTest to something like PerformanceTestCases, because we plan to add other cases to the same file.